### PR TITLE
Fix landing page on small screens.

### PR DESCRIPTION
**Pull request by borisnezlobin**
Fixed landing page on small screens (I don't have any before/after screenshots). Now, the TypeHero logo doesn't overlap top nav:
image
I also moved up the code images in the first feature card, but there is some weird behaviour while transitioning (don't merge yet).
Still WIP.

### CODE REVIEW

**Comment by bautistaaa** [comment]
can you run `pnpm checks:fix`

**Comment by borisnezlobin** [comment]
Will do! This is still a WIP, I have to work on the feature cards a bit more.

**Comment by bautistaaa** [comment]
you have conflicts cause we redid some things

File: apps/web/src/app/[locale]/_components/feature-card.tsx [hunk 1]
```diff
@@ -402,7 +402,7 @@ export function CollaborativeEnvironmentCard(props: CardProps) {
         >
           <div className="flex items-center gap-2">
             <Badge name="dax" />
-            <div className="text-xs text-neutral-500">5 years ago</div>
```
Comment (PickleNik) [inline apps/web/src/app/[locale]/_components/feature-card.tsx:405]: me thinks 5 years ago was more funnier

**Review by PickleNik — APPROVED** [review]

File: packages/ui/src/components/badge.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { cva, type VariantProps } from 'class-variance-authority';
 import { cn } from '../cn';
 
 const badgeVariants = cva(
-  'inline-flex items-center border rounded-full font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-0',
+  'inline-flex select-none items-center border rounded-full px-2.5 py-0.5 text-xs font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-0',
```
Comment (PickleNik) [inline packages/ui/src/components/badge.tsx:8]:
```suggestion
  'inline-flex select-none items-center border rounded-full px-2.5 py-0.5 font-semibold transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-0',
```
I'm not sure if `text-xs` is necessary here

File: apps/web/src/app/[locale]/tracks/_components/track-challenge-card.tsx [hunk 3]
```diff
@@ -131,7 +131,7 @@ export function MockTrackChallenge({ challenge }: { challenge: Challenge }) {
           <Check className="absolute left-1 my-auto h-3 w-3 scale-0 stroke-[4] text-white duration-300 peer-checked:scale-100 dark:text-black" />
           {challenge.name}
         </div>
-        <div className="hidden md:block">
+        <div className="md:block">
```
Comment (PickleNik) [inline apps/web/src/app/[locale]/tracks/_components/track-challenge-card.tsx:134]: I don't know if this is necessary, isn't it block by default? if we're removing hidden on mobile I don't think `md:block` is needed
```suggestion
        <div>
```

File: apps/web/src/app/[locale]/_components/feature-card.tsx (hunk 1 above)
Comment (borisnezlobin) [reply apps/web/src/app/[locale]/_components/feature-card.tsx:405]: yeah but it suggests an inactive community

File: apps/web/src/components/navigation.tsx
Comment (PickleNik) [inline apps/web/src/components/navigation.tsx:1]: pls undo, it should be camelCase

File: apps/web/src/components/navigation.tsx [hunk 4]
```diff
@@ -56,8 +56,8 @@ export function Navigation() {
                 >
                   <rect width="38" height="38" rx="4.5" fill="#3178C6" />
                   <path
-                    fillRule="evenodd"
-                    clipRule="evenodd"
+                    fill-rule="evenodd"
```
Comment (PickleNik) [inline apps/web/src/components/navigation.tsx:59]:
```suggestion
                    fillRule="evenodd"
```

File: apps/web/src/components/navigation.tsx [hunk 5]
```diff
@@ -56,8 +56,8 @@ export function Navigation() {
...
+                    clip-rule="evenodd"
```
Comment (PickleNik) [inline apps/web/src/components/navigation.tsx:60]:
```suggestion
                    clipRule="evenodd"
```

**Comment by PickleNik** [comment]
some things with new landing changes
localhost_3000_(iPhone 12 Pro) (20)
localhost_3000_(iPhone 12 Pro) (21)

**Comment by PickleNik** [comment]
> some things with new landing changes

fixed this elsewhere now, let's merge this PR as soon as main turns green

File: apps/web/src/app/[locale]/_components/feature-card.tsx [hunk 6]
```diff
@@ -73,7 +73,7 @@ function FeatureCard({
           bgClass,
         )}
       >
-        <div className="mx-10 my-10 min-h-[450px] w-full">
+        <div className="mx-10 my-10 min-h-[250px] w-full sm:min-h-[450px]">
```
Comment (PickleNik) [inline apps/web/src/app/[locale]/_components/feature-card.tsx:76]:
```suggestion
        <div className="mx-10 my-10 min-h-[250px] w-full md:min-h-[450px]">
```

File: apps/web/src/app/[locale]/_components/hero.tsx [hunk 7]
```diff
@@ -243,4 +243,4 @@ export async function Hero() {
       </div>
     </section>
   );
-}
+}
```
Comment (PickleNik) [inline apps/web/src/app/[locale]/_components/hero.tsx:246]:
```suggestion
}

```

**Review by PickleNik — CHANGES_REQUESTED** [review]

**Review by PickleNik — APPROVED** [review]

**Pull request merged**